### DISCUSSION TITLE
How to disable edge selection ?

### DISCUSSION THREAD

**Discussion by gitHoussem** [bcakmakoglu/vue-flow#1157] [Q&A] [answered]
Hi,

As you can see from the gif below, when trying to move around in the viewport, i can grab and move anywhere I want, until I reach an edge, when I try to move around when my cursor is on the edge, the edge selection event (I presume) is dominant, I can't move around until I put my cursor outside the edge. Is there a way to disable this? My edges should not be interactable at all (they are created and removed programmatically).

![edge select](https://github.com/bcakmakoglu/vue-flow/assets/43743206/8b866952-bc7b-41da-a5cc-1bec237e920c)

Thank you !

**Comment by bcakmakoglu** [comment, marked as answer]
You can set `edge.selectable` to false or just disable pointer events on edges altogether with css

```css
.vue-flow__edge {
	pointer-events: none;
}
```

**Reply by gitHoussem** [reply]
Hey man, thanks for the quick response. 
Both solutions worked like a charm! Props to you
